### ISSUE TITLE
Potential incompatibility with Death Rattle

### ISSUE THREAD

**Issue by Vectorial1024** [Vectorial1024/EliteBionicsFramework#84]
Steam user reports a problem:

> I've run into a strange issue: when used with Death Rattle Continued [1.2+] https://steamcommunity.com/sharedfiles/filedetails/?id=2896207870 , and, possibly, some other mod I can not detect, causes pawns to have their torso HP set to 8. some other body parts might be affected too, but I am 100% sure about torso only. Here's logs https://gist.github.com/HugsLibRecordKeeper/7ff5f7f31e2314a5af4f453afbf67d0d. Feel free to tag me if you need additional details about the issue. I am also notifing DRC[1.2+] author about the problem, in case it is on their end.

**Comment by kittycat2002** [comment]
I have just tested and it does not seem to occur with just Death Rattle and EBF (along with dependencies)

**Comment by Vectorial1024** [comment]
ref the log; this

> Failed to patch EdB.PrepareCarefully.PanelAppearance

---

then, ref #82

**Comment by Vectorial1024** [comment]
Ref the log file again; see line 3311 etc:

```
Failed to patch EdB.PrepareCarefully.PanelAppearance: System.Exception: Parameter "customPawn" not found in method System.Single EdB.PrepareCarefully.PanelAppearance::DrawColorSelectorForPawnLayer(EdB.PrepareCarefully.CustomizedPawn customizedPawn, System.Single cursorY, System.Collections.Generic.List`1<UnityEngine.Color> swatches, System.Boolean allowAnyColor)
[Ref 121E6694]
 at HarmonyLib.MethodPatcher.EmitCallParameter (System.Reflection.MethodInfo patch, System.Collections.Generic.Dictionary`2[TKey,TValue] variables, System.Reflection.Emit.LocalBuilder runOriginalVariable, System.Boolean allowFirsParamPassthrough, System.Reflection.Emit.LocalBuilder& tmpInstanceBoxingVar, System.Reflection.Emit.LocalBuilder& tmpObjectVar, System.Boolean& refResultUsed, System.Collections.Generic.List`1[T] tmpBoxVars) [0x00ada] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.MethodPatcher+<>c__DisplayClass36_0.<AddPostfixes>b__1 (System.Reflection.MethodInfo fix) [0x00006] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) [0x00014] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.MethodPatcher.AddPostfixes (System.Collections.Generic.Dictionary`2[TKey,TValue] variables, System.Reflection.Emit.LocalBuilder runOriginalVariable, System.Boolean passthroughPatches) [0x00040] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.MethodPatcher.CreateReplacement (System.Collections.Generic.Dictionary`2[System.Int32,HarmonyLib.CodeInstruction]& finalInstructions) [0x003d5] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.PatchFunctions.UpdateWrapper (System.Reflection.MethodBase original, HarmonyLib.PatchInfo patchInfo) [0x00059] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.PatchProcessor.Patch () [0x000fc] in <abec11463bc04855a5322a0a868aeb22>:0 
 at HarmonyLib.Harmony.Patch (System.Reflection.MethodBase original, HarmonyLib.HarmonyMethod prefix, HarmonyLib.HarmonyMethod postfix, HarmonyLib.HarmonyMethod transpiler, HarmonyLib.HarmonyMethod finalizer) [0x0002a] in <abec11463bc04855a5322a0a868aeb22>:0 
 at <0x15bff957850 + 0x001a2> <unknown method> 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 39)
```

This looks like the cause of the problem.

However, EBF does not patch `EdB.PrepareCarefully.PanelAppearance`, so it is not exactly EBF's problem. Probably some other mods.

Therefore, closing this.
